### PR TITLE
API: introduce getAmdCallback() to utils API (fixes #65)

### LINKED ISSUE #65: Introduce getAmdCallback() to utils API
It is used in `sort-amd-paths` and `amd-function-arity` rules.

### PR DESCRIPTION
**Pull request by obrejla**

### CODE REVIEW

File: lib/rules/amd-function-arity.js [hunk 1]
```diff
@@ -34,10 +34,6 @@ module.exports = function (context) {
         return node.arguments.filter(isFunctionExpression).length;
```
Comment (cvisco) [inline lib/rules/amd-function-arity.js:34]: would it make sense to move this out to lib/utils as well?

Comment (obrejla) [inline lib/rules/amd-function-arity.js:34]: Wouldn't it be enough to replace it by `util.isAmdRequire()`? But it's imho a different refactoring.

Comment (cvisco) [inline lib/rules/amd-function-arity.js:34]: Fair enough. Can be handled separately.

**Comment by cvisco** [comment]
@obrejla, merged. Thank you!

**Comment by obrejla** [comment]
You are welcome, thanks!

**Pull request merged**